lib/tasks/database_updates/migrate_uphold_access_parameters.rake

```diff
@@ -0,0 +1,58 @@
+require 'mongo'
+
+namespace :database_updates do
+  task :migrate_uphold_access_parameters => :environment do
+    index = 0
+    mongo_uri = `heroku config:get -a bat-eyeshade MONGODB_URI`.strip
+
+    Mongo::Logger.logger.level = Logger::FATAL
+
+    client = Mongo::Client.new(mongo_uri)
+    owners = client[:owners]
+
+    iterations = owners.count
+    puts "Anticipating queueing #{iterations} entries"
+
+    owners.find.each do |row|
```

Comment:
I believe this is `find_each`?

Reply:
Since this is a `Mongo::Collection` find_each doesn't work 😞 

Here's an example of how the Mongo connector works
https://github.com/mongolab/mongodb-driver-examples/blob/master/ruby/ruby-driver/ruby_simple_example.rb